src/Vlc.DotNet.Core/VideoManagement.cs

```diff
@@ -46,5 +48,15 @@ public string Deinterlace
         public IMarqueeManagement Marquee { get; private set; }
         public ILogoManagement Logo { get; private set; }
         public IAdjustmentsManagement Adjustments { get; private set; }
+
+        public bool IsMouseInputDisabled
+        {
+            set { myManager.SetMouseInput(myMediaPlayer, value); }
```

Comment:
Please check the logic here. If "disabled" is true, then "state" in SetMouseInput is `true` and `1` is passed, which means `on`. Unless I'm missing something, that logic is broken, but will be OK once you have performed the renaming above.

Reply:
Please note that "enabled" means "enable handling by libvlc"